tools/js2c.py

```diff
@@ -266,10 +273,7 @@ def js2c(buildtype, no_snapshot, js_modules, js_dumper, verbose=False):
 
         sorted_strings = sorted(magic_string_set, key=lambda x: (len(x), x))
         for idx, magic_string in enumerate(sorted_strings):
-            if not isinstance(magic_string, str):
-                magic = magic_string.decode('utf-8')
-            else:
-                magic = magic_string
+            magic = force_str(magic_string)
```

Comment:
Ohhh yes, that is true. We don't need to call the force_str here it's just a leftover from the first version. Will remove it asap.